Packs/FeedCognyteLuminar/Integrations/CognyteLuminar/CognyteLuminar.py

```diff
@@ -0,0 +1,476 @@
+    all_tags = set()
+    for children in childrens:
+        pattern = children.get("pattern")
+        for match in stix_regex_parser.findall(pattern):
+            stix_type, _, value = match
+            malware_types = parent.get("malwareTypes")
+            if stix_type in LUMINAR_TO_XSOAR_TYPES.keys():
+                indicator_type = LUMINAR_TO_XSOAR_TYPES[stix_type]
+            else:
+                indicator_type = stix_type
+            children.update(indicator_type=indicator_type)
+            parent["name"] = children["name"]
+            children["value"] = value
+            relationship_obj = EntityRelationship(
+                name=EntityRelationship.Relationships.INDICATED_BY,
+                entity_a=value,
+                entity_a_type=indicator_type,
+                entity_b=parent["name"],
+                entity_b_type=ThreatIntel.ObjectsNames.MALWARE,
+            )
+            tags = [it for it in children["indicator_types"]]
+            all_tags.update(tags)
+
+            tags.append("Malware Family: " + parent["name"])
+            malware_types_str = ",".join(malware_types)
+            indicator_type_str = ",".join(children["indicator_types"])
+            tags.append(f"Malware Type: {malware_types_str}")
+            if feed_tags:
+                tags.extend([ft for ft in feed_tags])
+
+            indicator = {
+                'value': value,
+                'occurred': datetime.strptime(children["created"],
+                                              '%Y-%m-%dT%H:%M:%S.%f%z').strftime(
+                    "%m/%d/%Y, %H:%M:%S"),
+                'type': indicator_type,
+                'rawJSON': dict(children),
+                'fields': {
+                    "category": "Malware",
+                    "malwarefamily": children["name"],
+                    "shortdescription": f"This {indicator_type} was involved in {indicator_type_str}",
+                    "creationdate": children["created"],
+                    'firstseenbysource': children["created"],
+                    "updateddate": children["modified"],
+                    'lastseenbysource': children["modified"],
+                    "tags": tags,
+                    "trafficlightprotocol": tlp_color
+                },
+                'relationships': [relationship_obj.to_indicator()]
+            }
+            modified_childrens.append(indicator)
+
+    if parent:
+        additional_fields = {
+            "STIX Is Malware Family": parent["is_family"],
+            "tags": list(all_tags), "stixid": parent["id"],
+            'STIX Malware Types': parent["malwareTypes"],
+            'malware_types': parent["malwareTypes"]
+        }
+        modified_parent = {
+            'value': parent["name"],
+            'occurred': datetime.strptime(parent["created"],
+                                          '%Y-%m-%dT%H:%M:%S.%f%z').strftime(
+                "%m/%d/%Y, %H:%M:%S"),
+            'type': "STIX Malware",
```

Comment:
@ChanochShayner this should be `Malware`, correct?

Reply:
Yes, same as here.
@yucohen please make sure it's all over the integration.

Reply:
@RamboV - This is still `STIX Malware` and not `Malware`